<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>480</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_buttons">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_buttons">
       <item>
        <widget class="QPushButton" name="pushButton_Start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Takeback">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Takeback</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Move_Now">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Move now</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_View_Log">
         <property name="text">
          <string>View Log</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Options">
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Quit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>455</y>
      <width>729</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>18</height>
     </size>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_estimating_now">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>14</height>
        </size>
       </property>
       <property name="text">
        <string>Estimating now: B1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_best_estimated">
       <property name="text">
        <string>Best estimated:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_expected_rest_time">
       <property name="text">
        <string>Expected rest time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_rest_time">
       <property name="text">
        <string>Rest time:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_player_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 0</string>
    </property>
    <widget class="QLabel" name="label_player_name_0">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vasya</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_chips_number_0">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_player_1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>141</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 1</string>
    </property>
    <widget class="QLabel" name="label_player_name_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Petya</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_chips_number_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionSave_game">
   <property name="text">
    <string>Save game</string>
   </property>
  </action>
  <action name="actionLoad_game">
   <property name="text">
    <string>Load game</string>
   </property>
  </action>
  <action name="actionSave_options">
   <property name="text">
    <string>Save options</string>
   </property>
  </action>
  <action name="actionLoad_options">
   <property name="text">
    <string>Load options</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionTakeback">
   <property name="text">
    <string>Takeback</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionMove_now">
   <property name="text">
    <string>Move now</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
